docs/source/simulation_study_results.rst

```diff
@@ -0,0 +1,317 @@
+Simulation studies
+==================
+
+This document is currently a work in progress. The results documented
+here originate from Gerbrich Ferdinands’ master’s thesis which can be
+found `here
+<https://github.com/GerbrichFerdinands/asreview-thesis/blob/master/manuscript/manuscript/Ferdinands%2C-G---MSBBSS.pdf/>`_.
+
+
+To provide insight in how much screening effort ASReview could
+potentially save, seven ASReview models were simulated on six existing
+systematic review datasets. In short, for all six datasets ASReview
+could have saved at least 60% of screening effort (e.g. WSS95% was <
+40%). For some datasets, ASReview was even able to detect 95% of
+relevant publications after screening only 5% of relevant publications.
+
+Datasets
+--------
+
+Datasets were collected from the fields of medicine (Cohen et al. 2006;
+Appenzeller‐Herzog et al. 2019), virology (Kwok et al. 2020), software
+engineering (Yu, Kraft, and Menzies 2018), behavioural public
+administration (Nagtegaal et al. 2019) and psychology (van de Schoot et
+al. 2017), to assess generalizability of the models across research
+contexts. Datasets are available in the `ASReview systematic review
+datasets
+repository <https://github.com/asreview/systematic-review-datasets>`__.
+
+Data were preprocessed from their original source into a test dataset,
```

Comment:
I think "test dataset" might be a little confusing as it might suggest cross validation, which is probably not what you mean.